documentation/tutorial/installing_the_diagnostics_tools.md

```diff
@@ -27,7 +27,7 @@ In order to generate core dumps for .net core apps, you can use the dotnet-dump
 > dotnet tool install --global dotnet-dump --version 1.0.4-preview6.19311.1
 > ```
 
-Please note, the dotnet-trace tool is version independent but as of right now (preview 5) it requires the 2.1 runtime to be installed to work. This will be fixed moving forward.
+Please note, the dotnet-dump tool is version independent but as of right now (preview 5) it requires the 2.1 runtime to be installed to work. This will be fixed moving forward.
```

Comment:
We are on Preview 6 now and it doesn't require the 2.1 SDK to be installed anymore. So you can remove the whole paragraph.

Reply:
Perhaps, but I do not have visibility of this fact as I do not contribute to the project, nor do I follow it, so I cannot assert this myself. I found an obvious error so submitted a PR to fix it. It seems the documentation is pretty dated, so this change is not relevant. Someone will likely update the documentation at some point to bring it up to date, so I'll abandon this PR as it is not relevant.
Thanks for the information.